types/global.d.ts

```diff
@@ -252,10 +252,17 @@ declare function Uint<N extends widths>(value: number | string): uint<N>;
 
 declare type ufixed<N extends widths, M extends precisions> = Brand<number, `ufixed${N}x${M}`>;
 
+declare class Byte {}
+
+declare class Byteslice<Length = void> {}
```

Comment:
`Byteslice` isn't user-facing is it? It feels like a weird classname, but if users always use `bytes` instead then it doesn't matter

Reply:
Yeah the user facing type is `bytes<N>`. I had to go with `Byteslice` here because I wanted `Bytes` for the tag function